homework-16/common/src/message.rs

```diff
@@ -0,0 +1,126 @@
+//! This module defines the message structures used for communication between the client and server.
+//! It includes the `ClientServerMessage` and `ServerClientMessage` structs.
+//
+//! All possible message types are defined in the `Payload` enum. This limits the extensibility
```

Comment:
you can wrap the backticked type names in `[]` and that will link to those items, so long as they can be resolved unambiguously